ktor-servlet/src/org/jetbrains/ktor/servlet/ServletApplicationResponse.kt

```diff
@@ -21,6 +23,7 @@ public class ServletApplicationResponse(private val servletResponse: HttpServlet
         override fun getHostHeaderNames(): List<String> = servletResponse.headerNames.toList()
         override fun getHostHeaderValues(name: String): List<String> = servletResponse.getHeaders(name).toList()
     }
+    override val cookies = ResponseCookies(this)
```

Comment:
Yes. The other reason to not use host cookie implementation for response is to reduce possible behaviour inconsistence among the hosts